File: Bioinformatics Stronghold/001_DNA/countDNAnucleotides.R
require(stringr)
require(readr)


countDNAnucleotides <- function(file) {
  DNA <- read_file(DNA)
  counts = c(str_count(DNA,"A"), str_count(DNA,"C"), str_count(DNA,"G"),str_count(DNA,"T"))
  return(counts)
}

file = "./rosalind_dna.txt"
countDNAnucleotides(file)

File: Bioinformatics Stronghold/001_DNA/countDNAnucleotides.py
def countDNAnucleotides(file):
	f = open(file, "r")
	DNA =f.read()
	counts = [DNA.count('A'), DNA.count('C'), DNA.count('G'), DNA.count('T')]
	return counts


if __name__ == "__main__":
	file = "rosalind_dna.txt"
	print(countDNAnucleotides(file))